### PR TITLE
Deployment script

### PR DESCRIPTION
**Pull request by dongbohu**
This PR includes a shell script (and related configuration files) that will deploy hetmech-backend API web server on a vanilla Ubuntu 18.04 box. I am using it to keep track of all deployment steps for now.

### CODE REVIEW

File: deployment/gunicorn.conf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[program:hetmech-gunicorn]
+command=/home/ubuntu/miniconda/envs/hetmech-backend/bin/gunicorn dj_hetmech.wsgi:application --bind 127.0.0.1:8001 --error-logfile /tmp/hetmech-gunicorn.log -w 3
```
Comment (dhimmel) [inline deployment/gunicorn.conf]: Is `-w` mean `--workers`?

**Review by dhimmel — COMMENTED** [review]
I am not sure in which order to run these files. Would a small `deployment/README.md` be helpful?

This is a good start towards scripting / automating the setup.

File: deployment/gunicorn.conf (hunk 1 above)
Comment (dongbohu) [reply deployment/gunicorn.conf]: Yes, we can also use `--workers=3`.

File: deployment/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Deployment
+
+This directory includes a shell script `setup.sh` and a few configuration files
+for packages that are required for hetmech backend deployment.
```
Comment (dhimmel) [inline deployment/README.md]: One sentence per line.

File: deployment/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Here is a summary of what this script does:
+ - Install/configure Nginx web server
+ - Install SSL certificate (issued by Let's Encrypt)
+ - Install a daily cron job to upgrade packages using `apt` command
+ - Install/configure Miniconda
+ - Download hetmech-backend code from Github and create a Conda environment
+ - Install/configure `supervisord`, which runs `Gunicorn` as a daemon
+
+Please reboot the deployment box at the end to ensure that the new
+configurations will become effective.
```
Comment (dhimmel) [inline deployment/README.md]: One sentence per line.

**Review by dhimmel — APPROVED** [review]
Looks good besides for two reflowed sentences (we use one sentence per line to track prose with git).

@dongbohu after fixing this, feel free to squash merge.

File: deployment/README.md (hunk 2 above)
Comment (dongbohu) [reply deployment/README.md]: Don't you think that the line will be a little too long?

**Pull request merged**